CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Added Features
 
 - Telemetry collection enabled by default for self-managed installations. Opt-out is available on bundle generation, or at any time via the System Configuration UI.
+- OpenShift Container Platform monitoring is configured and enabled by default for OpenShift 4 installations. Disabling is available on bundle generation, or at any time via the System Configuration UI.
```

Comment:
> System Configuration UI

There is a toggle in the operator CR UI, but not in the Central UI. I think we just should mention the Helm/CR flags here that may be used to disable it.